zvariant/src/value.rs

```diff
@@ -100,6 +101,22 @@ pub enum Value<'a> {
     Fd(Fd),
 }
 
+impl Eq for Value<'_> {}
+
+impl Ord for Value<'_> {
+    fn cmp(&self, other: &Self) -> Ordering {
+        self.partial_cmp(other)
+            .unwrap_or_else(|| match (self, other) {
+                (Self::F64(lhs), Self::F64(rhs)) => lhs.total_cmp(rhs),
+                // `partial_cmp` is returns `Some(_)` if either the discriminants are different
+                //   or if both the left hand side and right hand side is `Self::F64(_)`,
+                //   because `f64` is the only type in this enum, that does not implement `Ord`.
+                // This `match`-arm is therefore unreachable.
```

Comment:
Nice catch! I find it strange that the std API is making this a dynamic problem. In the case of `PartialEq` it's decided at compile time, what can and can't be compared. Anyway:

* A typo here: the first `is` can be dropped.
* while you're removing that anyway, might as well remove the extra space on line 112 and 113.

Reply:
:heavy_check_mark: